keyboards/splitography/rules.mk

```diff
@@ -0,0 +1,5 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = atmel-dfu
```

Comment:
```suggestion
BOOTLOADER = atmel-dfu

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = yes      # Enable Bootmagic Lite
MOUSEKEY_ENABLE = no        # Mouse keys
EXTRAKEY_ENABLE = no        # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
AUDIO_ENABLE = no           # Audio output
```

Reply:
These are  already defined in `info.json`, resulting in the following warnings if I had them to `rules.mk`:

```
⚠ splitography: Feature bootmagic is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature mousekey is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature extrakey is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature console is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature command is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature nkro is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature backlight is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature audio is specified in both info.json and rules.mk, the rules.mk value wins.
⚠ splitography: Feature steno is specified in both info.json and rules.mk, the rules.mk value wins.
```

Is it better to use `info.json` or rules.mk` for these?

Reply:
Ah, okay, I missed that.  It should be fine then.